distribution/pom.xml

```diff
@@ -154,18 +154,33 @@
             </executions>
          </plugin>
          <plugin>
-            <groupId>org.wildfly.build</groupId>
-            <artifactId>wildfly-server-provisioning-maven-plugin</artifactId>
+            <groupId>org.jboss.galleon</groupId>
+            <artifactId>galleon-maven-plugin</artifactId>
             <executions>
                <execution>
                   <id>provision-modules</id>
                   <goals>
-                     <goal>build</goal>
+                     <goal>provision</goal>
                   </goals>
                   <phase>compile</phase>
                   <configuration>
-                     <config-file>wildfly-provisioning.xml</config-file>
-                     <server-name>${infinispan.brand.prefix}-${wildfly.brand.prefix}-modules-${infinispan.brand.version}</server-name>
+                     <install-dir>${project.build.directory}/${infinispan.brand.prefix}-${wildfly.brand.prefix}-modules-${infinispan.brand.version}</install-dir>
+                     <record-state>false</record-state>
+                     <!-- true: uses local m2 cache only -->
+                     <offline>false</offline>
+                     <feature-packs>
+                        <!-- Infinispan modules -->
+                        <feature-pack>
+                           <groupId>${project.groupId}</groupId>
+                           <artifactId>infinispan-feature-pack</artifactId>
+                           <version>${project.version}</version>
+                           <inherit-configs>false</inherit-configs>
+                        </feature-pack>
+                     </feature-packs>
+                     <plugin-options>
+                        <!-- true to provision a thin installation, false to provision a fat installation. -->
+                        <jboss-maven-dist>false</jboss-maven-dist>
```

Comment:
Don't we want a "thin" distribution here so that the user only gets the Infinispan related modules?

I haven't caught up with Galleon yet.

Reply:
I'll change the comment :) I'm just abusing the words "thin" and "fat" from the previous server

when `false`, the plugin downloads the jars 
when `true`, the module.xml is populated with the maven coordinates and WF downloads them from maven central (or any repository is settings.xml) on startup.

Reply:
updated :)

Reply:
Thanks, that's much clearer :slightly_smiling_face: